### PR TITLE
added: pass render system to use to add-ons

### PR DESCRIPTION
**Pull request by notspiff**
wayland can use either gl or gles and we need to know which in add-ons

### CODE REVIEW

File: tools/depends/configure.ac [hunk 1]
```diff
@@ -97,6 +97,10 @@ AC_ARG_ENABLE([wayland],
   [enable_wayland=$enableval],
   [enable_wayland=no])
 
+AC_ARG_WITH([rendersystem],
+  [AS_HELP_STRING([--with-rendersystem],
+  [render system to use])],
+  [app_rendersystem=$withval])
```
Comment (wsnipex) [inline tools/depends/configure.ac]: we should probably add a content sanity check for this option

Comment (notspiff) [reply tools/depends/configure.ac]: just sanity for gl vs gles or actually checking if value is valid for platform in question ?

Comment (wsnipex) [reply tools/depends/configure.ac]: gl/gles check should be enough imho

Comment (notspiff) [reply tools/depends/configure.ac]: added.

**Review by wsnipex — APPROVED** [review]

File: cmake/scripts/windows/ArchSetup.cmake [hunk 2]
```diff
@@ -12,6 +12,7 @@ endif()
 # -------- Paths (mainly for find_package) ---------
 
 set(PLATFORM_DIR platform/win32)
+set(APP_RENDER_SYSTEM dx11)
```
Comment (afedchin) [inline cmake/scripts/windows/ArchSetup.cmake]: please add the same for `cmake/scripts/windowsstore/ArchSetup.cmake`

**Review by afedchin — CHANGES_REQUESTED** [review]

File: cmake/scripts/windows/ArchSetup.cmake (hunk 2 above)
Comment (notspiff) [reply cmake/scripts/windows/ArchSetup.cmake]: done, thought they shared stuff..

**Review by afedchin — APPROVED** [review]
thanks

File: cmake/scripts/windows/ArchSetup.cmake (hunk 2 above)
Comment (afedchin) [reply cmake/scripts/windows/ArchSetup.cmake]: thanks, not all things are shared.

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
jenkins build this please

**Pull request merged**